src/pages/_app.tsx

```diff
@@ -27,7 +28,7 @@ import type { AppProps } from "next/app";
 const { chains, provider } = configureChains(
   [CHAIN],
   [
-    alchemyProvider({ apiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY || "" }),
+    alchemyProvider({ apiKey: env.NEXT_PUBLIC_ALCHEMY_API_KEY || "" }),
```

Comment:
Since all environment variables are now validated and defined, we can probably remove `|| ""` everywhere right?

Reply:
I think that leaving it like this may come in handy in case a build is launched with 'SKIP_ENV_VALIDATION=true', to prevent it from breaking under such circumstances